scripts/bump-holochain.sh

```diff
@@ -29,7 +29,7 @@ CURRENT_BRANCH=${1:-$(git branch --show-current)}
 SEARCH_PATTERN="no-tag"
 case $CURRENT_BRANCH in
     main)
-        SEARCH_PATTERN="^holochain-0.6.[0-9]+-dev.[0-9]+$"
+        SEARCH_PATTERN="^holochain-0.6.[0-9]+-.*.[0-9]+$"
```

Comment:
Note that this should probably actually be:

```suggestion
        SEARCH_PATTERN="^holochain-0\.6\.[0-9]+-.*\.[0-9]+$"
```

But that change should be made on the other search strings too.